app/src/main/java/com/hover/stax/domain/use_case/sims/ListSimsUseCase.kt

```diff
@@ -0,0 +1,45 @@
+package com.hover.stax.domain.use_case.sims
+
+import com.hover.sdk.actions.HoverAction
+import com.hover.sdk.sims.SimInfo
+import com.hover.stax.data.local.SimRepo
+import com.hover.stax.data.local.actions.ActionRepo
+import com.hover.stax.domain.model.Account
+import com.hover.stax.domain.repository.AccountRepository
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.withContext
+import timber.log.Timber
+
+data class SimWithAccount(
+    val sim: SimInfo,
+    val account: Account,
+    val balanceAction: HoverAction?,
+    val airtimeAction: HoverAction?
+)
+
+class ListSimsUseCase(
```

Comment:
This is what a use case is good for! It accesses multiple separate repos puts the data together for use by the view model and ui. It handles account creation if needed: the viewmodel and ui shouldn't have anything do with this since it isn't based on the user's actions